.appveyor.yml

```diff
@@ -4,27 +4,14 @@ init:
 environment:
   matrix:
   - QT_BASE_DIR: C:\Qt\5.6\mingw49_32
-    MINGW_BASE_DIR: C:\Qt\Tools\mingw492_32
-    MINGW_BASE_DIR_BASH: /c/Qt/Tools/mingw492_32
-#  - QT_BASE_DIR: C:\Qt\5.6\mingw49_32
```

Comment:
Um, these commented out build matrix entries will test code blocks that we do not _normally_ use but which may need retesting in future - I'd be reluctant to lose the record of the elements that control them `WITH_FONTS: NO` and `WITH_UPDATER: NO` ...

Reply:
It's pretty easy to re-do if we need it though, no more than 5mins to copy/paste the lines and edit them...